src/relstorage/_util.py

```diff
@@ -604,3 +608,12 @@ def __repr__(self):
             self.needs_fetchall,
             self.last_stmt,
         )
+
+
+
+try:
```

Comment:
A comment about why this dance is necessary would be helpful.

Reply:
Added a comment with links to the python bug tracker